test/itex/test_tensorflow_itex_basic.py

```diff
@@ -217,5 +217,46 @@ def test_depthwiseconv2d_case(self):
                     reshape_counter += 1
             self.assertEqual(reshape_counter, 2)
 
+    @disable_random()
+    @unittest.skipIf(version1_lt_version2(tf.version.VERSION, '2.8.0'), "Only supports tf greater 2.7.0")
+    def test_itex_benchmark_gpu(self):
+        x = tf.compat.v1.placeholder(tf.float32, [1, 56, 56, 16], name="input")
+        top_relu = tf.nn.relu(x)
+        paddings = tf.constant([[0, 0], [1, 1], [1, 1], [0, 0]])
+        x_pad = tf.pad(top_relu, paddings, "CONSTANT")
+        conv_weights = tf.compat.v1.get_variable("weight", [3, 3, 16, 16],
+                                                 initializer=tf.compat.v1.random_normal_initializer())
+        conv = tf.nn.conv2d(x_pad, conv_weights, strides=[1, 2, 2, 1], padding="VALID")
+        normed = tf.compat.v1.layers.batch_normalization(conv)
+        conv_weights2 = tf.compat.v1.get_variable("weight2", [3, 3, 16, 16],
+                                                  initializer=tf.compat.v1.random_normal_initializer())
+        conv2 = tf.nn.conv2d(top_relu, conv_weights2, strides=[1, 2, 2, 1], padding="SAME")
+        normed2 = tf.compat.v1.layers.batch_normalization(conv2)
+        add = tf.raw_ops.Add(x=normed, y=normed2, name='addv2')
+        relu = tf.nn.relu(add)
+        relu6 = tf.nn.relu6(relu, name='op_to_store')
+        out_name = relu6.name.split(':')[0]
+        with tf.compat.v1.Session() as sess:
+            sess.run(tf.compat.v1.global_variables_initializer())
+            output_graph_def = graph_util.convert_variables_to_constants(
+                sess=sess,
+                input_graph_def=sess.graph_def,
+                output_node_names=[out_name])
+
+            quantizer = Quantization('fake_yaml_2.yaml')
+            dataset = quantizer.dataset('dummy', shape=(100, 56, 56, 16), label=True)
+            quantizer.eval_dataloader = common.DataLoader(dataset)
+            quantizer.calib_dataloader = common.DataLoader(dataset)
+            quantizer.model = output_graph_def
+            output_graph = quantizer.fit()
+
+            evaluator = Benchmark('fake_yaml_2.yaml')
+            evaluator.b_dataloader = common.DataLoader(dataset)
+            evaluator.model = output_graph
+            evaluator('performance')
+
+            self.assertEqual(os.environ.get('NC_ENV_CONF'), 'True')
```

Comment:
This env will be set as true no matter for single instance and multi-instance. For multi-instance it will be set after config_instance() done.

Reply:
It may be a solution to check "*.log" to identify whether go "config_instance" or not.